source/includes/code-examples/DeleteOneSync.cs

```diff
@@ -0,0 +1,70 @@
+using MongoDB.Bson;
+using MongoDB.Bson.Serialization.Attributes;
+using MongoDB.Bson.Serialization.Conventions;
+using MongoDB.Driver;
+using static System.Console;
+
+namespace CSharpExamples.UsageExamples;
+
+public class DeleteOneSync
```

Comment:
Change this class name and the filename to be DeleteOne rather than DeleteOneSync. This will make it match conventions we are using across the other usage examples